src/Time/Time.js

```diff
@@ -5,6 +5,32 @@ const INVALID = 'is-invalid';
 const CLOCK = ['am', 'pm'];
 
 class TimeItem extends Component {
+    static propTypes = {
+        arialabel: PropTypes.string,
+        disabled: PropTypes.bool,
+        downButtonProps: PropTypes.object,
+        format12Hours: PropTypes.bool,
+        inputProps: PropTypes.object,
+        max: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+        name: PropTypes.string,
+        placeholder: PropTypes.string,
+        spinners: PropTypes.bool,
+        style: PropTypes.string,
+        time: PropTypes.object,
+        type: PropTypes.string,
+        upButtonProps: PropTypes.object,
+        updateTime: PropTypes.func,
+        value: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
+    };
+    static defaultProps = {
+        arialabel: '',
+        disabled: false,
+        format12hours: false,
+        id: '',
+        spinners: false,
+        value: null
+    };
+
```

Comment:
I believe these were moved to a static reference outside the class to match all other components.